### DISCUSSION TITLE
brew audit errors

### DISCUSSION THREAD

**Discussion by jsupun** [Homebrew/discussions#3441] [Tap maintenance and Homebrew development] [answered]
I'm working on an automated build process for our products (minus PR creation). And to make sure all the checkboxes in the PR message are covered I'm doing a `brew audit --new our-product` which is returning.

```shell
Warning: audit is a developer command, so
Homebrew's developer mode has been automatically turned on.
To turn developer mode off, run brew developer off

Error: 2 problems in 1 formula detected
our-product:
  * HEAD: Use `branch: "master"` to specify the default branch
  * GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
```

At this point the Ruby script is commit, in a branch, in a fork of `Homebrew/homebrew-core`. It made it pass `brew test-bot` and `brew install --build-from-source` without any errors.

Does audit not like that the formula is in a branch and not in `Homebrew/homebrew-core`? Any way to get around this?

Thanks.

**Comment by SMillerDev** [comment, marked as answer]
It's refering to your `head` statement missing a branch name.

**Reply by jsupun** [reply]
Thanks, that fixed that one.
